tee-worker/litentry/core/stf-task/receiver/src/lib.rs

```diff
@@ -165,35 +167,60 @@ pub fn run_stf_task_receiver<K, A, S, H, O>(
 	context: Arc<StfTaskContext<K, A, S, H, O>>,
 ) -> Result<(), Error>
 where
-	K: ShieldingCryptoDecrypt + ShieldingCryptoEncrypt + Clone,
-	A: AuthorApi<Hash, Hash>,
-	S: StfEnclaveSigning,
-	H: HandleState,
+	K: ShieldingCryptoDecrypt + ShieldingCryptoEncrypt + Clone + Send + Sync + 'static,
+	A: AuthorApi<Hash, Hash> + Send + Sync + 'static,
+	S: StfEnclaveSigning + Send + Sync + 'static,
+	H: HandleState + Send + Sync + 'static,
 	H::StateT: SgxExternalitiesTrait,
-	O: EnclaveOnChainOCallApi + EnclaveMetricsOCallApi,
+	O: EnclaveOnChainOCallApi + EnclaveMetricsOCallApi + 'static,
 {
 	let receiver = stf_task_sender::init_stf_task_sender_storage()
 		.map_err(|e| Error::OtherError(format!("read storage error:{:?}", e)))?;
 
+	let (sender, to_receiver) = std::sync::mpsc::channel::<(ShardIdentifier, H256, TrustedCall)>();
+
+	// Spawn thread to handle received tasks
+	let context_for_thread = context.clone();
+	std::thread::spawn(move || loop {
+		if let Ok((shard, hash, call)) = to_receiver.recv() {
+			error!("Submitting trusted call to the pool");
```

Comment:
Why this is an `error` ?

Reply:
Updated to `info`

Reply:
@felixfaisal can this be `debug` or `trace`

Imho this log entry doesn't provide any useful information and will only spawn logs.